torch/_inductor/graph.py

```diff
@@ -649,6 +666,8 @@ def compile_to_module(self):
         from .codecache import PyCodeCache
 
         code, linemap = self.codegen()
+        if config.debug:
```

Comment:
please remove this, or the generated code will be printed twice (it is printed on line 677)
Also, use standard logging facilities in `compile_to_fn` instead of `if config.debug: print...`